Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -36,7 +36,7 @@ private struct TabLocationViewUX {
     static let URLBarPadding = 4
 }
 
-class TabLocationView: UIView {
+class TabLocationView: UIView, FeatureFlagsProtocol {
```

Comment:
@electricRGB do we need this to have protocol, I don't see feature implementation here?

Reply:
Maybe its used somewhere else so want to confirm.

Reply:
oops. Forgot to remove it from here and one other location. Updated now.